### DISCUSSION TITLE
New project structure

### DISCUSSION THREAD

**Discussion by PavelBal** [esqLABS/esqlabsR#564] [Ideas]
- On GitHub/SVN
- Observed data global for whole project, not for versions
- #577

**Comment by Felixmil** [comment]
Make ProjectConfiguration interoperable between: PKSim/Mobi export / Json Files / Shiny editing app.

**Comment by PavelBal** [comment]
- For all excel files - append the current version (e.g., "V00.08") to the name of the file, in order to be able to open the same file (e.g. "Parameters") from different versions for comparison.

**Comment by PavelBal** [comment]
- [ ] Rename projectConfiguration$paramsFile by projectConfiguration$modelParamsFile #558
- [ ] Rename PopulationParameters.xlsx to Populations.xlsx #554

**Comment by PavelBal** [comment]
- [ ] Idea: Make projectConfiguration object contain _ALL_ the project's parameters #560
